pkg/login/kubeConfig.go

```diff
@@ -35,23 +35,20 @@ func CreateClusterKubeConfig(clusterID string, kubeConfig api.Config) (string, e
 		}
 	}
 
-	// Write kube config if file not exist
+	// Write kube config
 	filename := filepath.Join(path, "config")
-	_, err = os.Stat(filename)
-	if errors.Is(err, os.ErrNotExist) {
-		f, err := os.Create(filename)
-		if err != nil {
-			return "", err
-		}
-		err = clientcmd.WriteToFile(kubeConfig, f.Name())
+	f, err := os.Create(filename)
+	if err != nil {
+		return "", err
+	}
+	err = clientcmd.WriteToFile(kubeConfig, f.Name())
 
-		if err != nil {
-			return "", err
-		}
-		err = f.Close()
-		if err != nil {
-			return "", err
-		}
+	if err != nil {
+		return "", err
+	}
+	err = f.Close()
```

Comment:
**Suggestion:** use defer to call `f.Close()`. 
For the error handling: https://trstringer.com/golang-deferred-function-error-handling/

Reply:
Yeah, I also thik using defer for calling `f.Close()` is better, it ensures that the file is closed as soon as the function `CreateClusterKubeConfig` returns, regardless of where the return happens.

Reply:
Done in latest commit...